<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=us-ascii">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="index_files/filelist.xml">
<title>PINCAGE homepage</title>
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Michal Switnicki</o:Author>
  <o:LastAuthor>Michal Switnicki</o:LastAuthor>
  <o:Revision>58</o:Revision>
  <o:TotalTime>1018</o:TotalTime>
  <o:Created>2015-06-11T00:23:00Z</o:Created>
  <o:LastSaved>2015-06-11T21:29:00Z</o:LastSaved>
  <o:Pages>2</o:Pages>
  <o:Words>1383</o:Words>
  <o:Characters>8304</o:Characters>
  <o:Lines>69</o:Lines>
  <o:Paragraphs>19</o:Paragraphs>
  <o:CharactersWithSpaces>9668</o:CharactersWithSpaces>
  <o:Version>14.00</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=dataStoreItem href="index_files/item0006.xml"
target="index_files/props007.xml">
<link rel=themeData href="index_files/themedata.thmx">
<link rel=colorSchemeMapping href="index_files/colorschememapping.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:Zoom>96</w:Zoom>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:HyphenationZone>21</w:HyphenationZone>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>PL</w:LidThemeOther>
  <w:LidThemeAsian>X-NONE</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SplitPgBreakAndParaMark/>
  </w:Compatibility>
  <w:BrowserLevel>MicrosoftInternetExplorer4</w:BrowserLevel>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="267">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;
	mso-font-charset:2;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:0 268435456 0 0 -2147483648 0;}
@font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;
	mso-font-charset:2;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:0 268435456 0 0 -2147483648 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
h1
	{mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Heading 1 Char";
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	mso-pagination:widow-orphan;
	mso-outline-level:1;
	font-size:24.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	font-weight:bold;}
h3
	{mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Heading 3 Char";
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	mso-pagination:widow-orphan;
	mso-outline-level:3;
	font-size:13.5pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	font-weight:bold;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	text-decoration:underline;
	text-underline:single;}
p
	{mso-style-priority:99;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 1";
	mso-ansi-font-size:14.0pt;
	mso-bidi-font-size:14.0pt;
	font-family:"Cambria","serif";
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#365F91;
	mso-themecolor:accent1;
	mso-themeshade:191;
	font-weight:bold;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-noshow:yes;
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 3";
	mso-ansi-font-size:12.0pt;
	mso-bidi-font-size:12.0pt;
	font-family:"Cambria","serif";
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#4F81BD;
	mso-themecolor:accent1;
	font-weight:bold;}
span.apple-converted-space
	{mso-style-name:apple-converted-space;
	mso-style-unhide:no;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-size:10.0pt;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;}
@page WordSection1
	{size:595.3pt 841.9pt;
	margin:72.0pt 981.6pt 72.0pt 72.0pt;
	mso-header-margin:35.4pt;
	mso-footer-margin:35.4pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 @list l0
	{mso-list-id:136193431;
	mso-list-template-ids:-2132233196;}
@list l0:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l0:level2
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:"Courier New";
	mso-bidi-font-family:"Times New Roman";}
@list l0:level3
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l0:level4
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l0:level5
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l0:level6
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l0:level7
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l0:level8
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l0:level9
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman","serif";}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="1026"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=PL link=blue vlink=purple style='tab-interval:35.4pt'>

<div class=WordSection1>

<h1 align=center style='margin-right:177.75pt;text-align:center'><span
lang=EN-GB style='mso-fareast-font-family:"Times New Roman";mso-ansi-language:
EN-GB'>PINCAGE - </span><span lang=EN-GB style='mso-ansi-language:EN-GB'>Probabilistic
INTegration of CAncer GEnomics data</span><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-GB'><o:p></o:p></span></h1>

<h3><span lang=EN-GB style='mso-ansi-language:EN-GB'>Overview</span><span
lang=EN-GB style='mso-fareast-font-family:"Times New Roman";mso-ansi-language:
EN-GB'><o:p></o:p></span></h3>

<p style='margin-right:177.75pt'><span lang=EN-GB style='mso-ansi-language:
EN-GB;mso-bidi-font-weight:bold'>PINCAGE is a method that uses probabilistic
integration of cancer genomics data for combined evaluation of RNA-seq gene
expression and 450K array DNA methylation measurements of promoters as well as
gene bodies.</span><span lang=EN-GB style='mso-ansi-language:EN-GB'>The method
learns the specific relationships between the data types and exploits these for
biomarker discovery and classification of new samples. It also explicitly
models the uncertainty of both the count-based NGS data and the continuous
array data measurements. This approach is specifically tailored for cancer
studies where much heterogeneity is observed among tumours. The method combines
graphical model formalism with non-parametric specification of probability distributions
to capture the highly context-specific relationships between methylation
patterns and gene expression.<o:p></o:p></span></p>

<h3><span lang=EN-GB style='mso-ansi-language:EN-GB'><section>Instructions</span><span
lang=EN-GB style='mso-fareast-font-family:"Times New Roman";mso-ansi-language:
EN-GB'><o:p></o:p></span></h3>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>PINCAGE was implemented using R (3.0.1+) and is
currently available as scripts governing the data preparation, model training
and evaluation. It relies on external binary available from &#8220;phy&#8221;
C++ library. We provide a statically compiled binary <a href="dfgEval_static">dfgEval_static</a>,
compiled for Ubuntu 12.04.5 LTS (GNU/Linux 3.2.0-75-generic x86_64). If your system
does not permit you to use that static binary, you may compile the phy library
as described here: <a href="http://github.com/jakob-skou-pedersen/phy/">http://github.com/jakob-skou-pedersen/phy/</a>,
and use the compiled ./phy/bin/dfgEval instead. At this point I ask you to
rename the binary to dfgEval_static and place in the directory with remaining R
scripts and data. Our scripts also require the following libraries to be
installed in your R system: <a
href="http://cran.r-project.org/web/packages/aws/index.html">aws</a> and <a
href="http://cran.r-project.org/web/packages/smoothie/index.html">smoothie</a>.
If you plan to re-run the simulated data analysis, you will additionally need <a
href="http://cran.r-project.org/web/packages/MASS/index.html">MASS</a> library.<o:p></o:p></span></p>

<h3 style='margin-right:177.75pt'><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-GB'>Data download<o:p></o:p></span></h3>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>We make example of PINCAGE with Breast Invasive
Carcinoma <a
href="https://tcga-data.nci.nih.gov/tcga/tcgaCancerDetails.jsp?diseaseType=BRCA&amp;diseaseName=Breast%20invasive%20carcinoma">dataset</a>
from The Cancer Genome Atlas. The processed data files are split by the
analyses: <a href="BRCA_TvAN/essentials_allBRCA.RData">normal vs tumour</a> and
<a href="BRCA_aggressive/essentials_aggressive.RData">progressing vs
non-progressing</a>. Download these .RData files for running the model. They
contain processed data in formats expected by the script files.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>To run the scripts below, enter in your prompt:<br>
</span><strong><span lang=EN-GB style='mso-ansi-language:EN-GB'>$ Rscript
scriptName ID_start ID_end<o:p></o:p></span></strong></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>The scripts are set up for running on HPC systems and
hence rely on numerical IDs for genes. Each script can process IDs in given succession
from </span><strong><span lang=EN-GB style='mso-ansi-language:EN-GB'>start_ID </span></strong><span
lang=EN-GB style='mso-fareast-font-family:"Times New Roman";mso-ansi-language:
EN-GB'>to </span><strong><span lang=EN-GB style='mso-ansi-language:EN-GB'>end_ID</span></strong><span
lang=EN-GB style='mso-fareast-font-family:"Times New Roman";mso-ansi-language:
EN-GB'>. To run the command for a single ID, set start and end to be the same
numbers. If you want to find the list of 17728 genes that were suitable for
PINCAGE modelling, load any of the provided .RData files and see the names of
genes in the workingList_BRCA vector.<o:p></o:p></span></p>

<h3 style='margin-right:177.75pt'><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-GB'>Scripts for BRCA normal vs tumour
comparison<o:p></o:p></span></h3>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>The BRCA tumour (G2) vs normal (G1) analysis was
performed with 2/3 of the data set using either single data type sub-models or
the integrative model. Additionally, predictions were made using the
integrative PINCAGE on the remaining BRCA tumours and normal. The following
scripts are available:<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="BRCA_TvAN/PINCAGE-expression_eval.R">PINCAGE-expression_eval.R</a>:
evaluation using gene expression PINCAGE sub-model<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="BRCA_TvAN/PINCAGE-geneBody_eval.R">PINCAGE-geneBody_eval.R</a>:
evaluation using gene body methylation PINCAGE sub-model<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="BRCA_TvAN/PINCAGE-promoter_eval.R">PINCAGE-promoter_eval.R</a>:
evaluation using promoter methylation PINCAGE sub-model<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="BRCA_TvAN/PINCAGE-integrative_eval.R">PINCAGE-integrative_eval.R</a>:
evaluation using integrative PINCAGE model. This script also produces model
files for use with the prediction script:<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="BRCA_TvAN/PINCAGE-integrative_predict.R">PINCAGE-integrative_predict.R</a>:
produces an output with posterior probabilities for the 1/3 of left out BRCA
dataset.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>The following is an example output which contains 4
lines:<o:p></o:p></span></p>

<p class=MsoNormal><strong><span lang=EN-GB style='mso-ansi-language:EN-GB'>$ head
1.result<o:p></o:p></span></strong></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>2.18940699622378e-307 60.8660000000091
1.85376000000571 1.5754059866035 37.458433255818<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>0 1.26170400928105 1.76036812802714 2.25370085162364
2.73682905236275 3.16286316599919 3.64057425338269 4.04940757249639
4.4294496481436 4.86153688417511 5.32437204069118 5.82180088987547
6.31358459692797 6.80504171869964 7.36508705450302 8.06933729509091
8.89610027687383 9.63792988608681 10.3480857295364 11.0540271381453
12.2252022238576 14.3937423746706 16.476045600892 19.4155215191607
28.1912320879913 4980.8466724282<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>-7.01 -6.02 -5.52 -5.175 -4.9 -4.655 -4.43 -4.2 -3.915
-3.315 -0.32 0.77 1.29 1.655 1.93 2.15 2.345 2.515 2.665 2.805 2.935 3.07 3.215
3.385 3.61 7.01<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>-7.01 0.13 0.61 0.95 1.23 1.48 1.715 1.935 2.15 2.35
2.545 2.75 2.965 3.165 3.36 3.535 3.68 3.815 3.945 4.08 4.22 4.375 4.56 4.795
5.13 7.01<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>The first line is the most important for you. The
first number of that line is the Z-test p-value. The second number is the value
of the D statistic (from Likelihood Ratio Test). The third and fourth numbers
are the mean and standard deviation of the random expectation of the D
statistic. The fifth number is the Z-score for given gene. In the lines below,
the discretization boundaries are output for normalized gene expression (2<sup>nd</sup>
line), gene body methylation (3<sup>rd</sup> line) and promoter methylation (4<sup>th</sup>
line). Additionally, the script produces a .tar file that contains trained
model specification that can be used with the PINCAGE-integrative_predict.R
script. The prediction script will take the tar file, unpack it and predict for
the remaining 1/3 of BRCA dataset to produce a table with sample IDs, tumour
posterior probabilities (posterior_G2) and &#8211;log-likelihoods (under G1 and
G2 models):<o:p></o:p></span></p>

<p class=MsoNormal><strong><span lang=EN-GB style='mso-ansi-language:EN-GB'>$ head
1.predicted<o:p></o:p></span></strong></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>sample_ID posterior_G2 G2_mloglik G1_mloglik<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-A7-A0D9-11A 0.310025518872388 108.23 107.43<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-A7-A0DB-11A 0.294422667408569 100.18 99.306<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-AC-A2FB-11A 0.394126331568238 128.65 128.22<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-AC-A2FM-11B 0.725119497789823 131.11 132.08<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-BH-A0AU-11A 0.455121107626422 111.05 110.87<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-BH-A0AY-11A 0.413382421082671 120.78 120.43<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-BH-A0B3-11B 0.495000166660001 122.21 122.19<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-BH-A0B8-11A 0.293177778906433 109.88 109<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-BH-A0BA-11A 0.447692090425673 133.68 133.47<o:p></o:p></span></p>

<h3 style='margin-right:177.75pt'><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-GB'>Scripts for BRCA progressing vs non-progressing
comparison<o:p></o:p></span></h3>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>The BRCA 14 progressing (G1) vs 57 non-progressing (G2)
tumours analysis was performed using the integrative model in the
cross-validation fashion. Therefore, the provided script performs 14-fold
cross-evaluation and validation at once. The following script is available:<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a
href="BRCA_aggressive/PINCAGE-integrative_evalANDpredict_LOU.R">PINCAGE-integrative_evalANDpredict_LOU.R</a><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>The example output contains gene Z-scores in each fold
of the validation procedure in the first line, and a table with &#8211;log-likelihoods
of each sample according to the cross-validation G1 and G2 models:<o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span lang=EN-GB
style='mso-fareast-font-family:"Times New Roman";mso-ansi-language:EN-GB'>$ head
1.result<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>1.59 1.44 1.58 2.60 2.20 0.98 1.30 1.87 1.02 2.12 2.56
1.43 1.62 4.22<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>sample_ID G1_mloglik G2_mloglik<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-A7-A3RF 108.63 108.73<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-A7-A425 94.815 94.879<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-LL-A5YM 106.13 106.18<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-E9-A243 106.07 105.98<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-A7-A13G 92.953 92.995<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-A7-A26H 98.799 98.934<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-LQ-A4E4 105.64 105.69<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>TCGA-A7-A13H 88.738 88.796<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>If you wish to repeat the full progression set analysis,
you must complete analysis of all 17728 genes using the above script and join
the respective sample likelihoods according to the ranking using Z-score. <o:p></o:p></span></p>

<h3 style='margin-right:177.75pt'><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-GB'>Scripts for data simulation and
analysis<o:p></o:p></span></h3>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>We also evaluated PINCAGE and its sub-models on
simulated datasets. We provide the following scripts for simulating the
analysed data sets:<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="simCodes/sim_dilution_true.R">sim_dilution_true.R</a>
script for simulating true positive genes of the tumour heterogeneity set<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="simCodes/sim_dilution_false.R">sim_dilution_false.R</a>
script for simulating true negative genes of the tumour heterogeneity set<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="simCodes/sim_deltaCorelation_true.R">sim_deltaCorelation_true.R</a>
script for simulating true positive genes of the delta correlation set<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="simCodes/sim_deltaCorelation_false.R">sim_deltaCorelation_false.R</a>
script for simulating true positive genes of the delta correlation set<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>And the corresponding sub-model and integrative PINCAGE
scripts for analysis of such produced data sets:<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="simCodes/PINCAGE-expression_eval_simData.R">PINCAGE-expression_eval_simData.R</a>
evaluation using gene expression simulated data only with the respective PINCAGE
sub-model<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="simCodes/PINCAGE-geneBody_eval_simData.R">PINCAGE-geneBody_eval_simData.R</a>
evaluation using gene body methylation simulated data only with the respective PINCAGE
sub-model<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="simCodes/PINCAGE-promoter_eval_simData.R">PINCAGE-promoter_eval_simData.R</a>
evaluation using promoter methylation simulated data only with the respective PINCAGE
sub-model<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'><a href="simCodes/PINCAGE-integrative_eval_simData.R">PINCAGE-integrative_eval_simData.R</a>
evaluation using all data types with the integrative PINCAGE model<o:p></o:p></span></p>

<h3 style='margin-right:177.75pt'><a name=reference></a><a name=instructions></a><span
style='mso-bookmark:reference'><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-GB'>Contact<o:p></o:p></span></span></h3>

<p style='margin-right:177.75pt'><span style='mso-bookmark:reference'><span
lang=EN-GB style='mso-ansi-language:EN-GB'>Should you have questions, please
contact: </span></span><span lang=EN-GB style='mso-ansi-language:EN-GB'>michal.switnicki
[at] gmail.com<o:p></o:p></span></p>

<h3>Reference<span lang=EN-US style='mso-ansi-language:EN-US'><o:p></o:p></span></h3>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'>PINCAGE has
been described in the following manuscript:<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'>Micha&#322;
P. &#346;witnicki, Malene Juul, Tobias Madsen, Karina D. S&oslash;rensen, Jakob
S. Pedersen <b style='mso-bidi-font-weight:normal'>PINCAGE: Probabilistic
integration of cancer genomics data for perturbed gene identification and
sample classification</b></span><span class=apple-converted-space><span
lang=EN-US style='font-size:13.5pt;color:black;mso-ansi-language:EN-US'>&nbsp;</span></span><em><span
lang=EN-US style='font-size:13.5pt;color:black;mso-ansi-language:EN-US'>(Submitted.)</span></em><span
lang=EN-US style='mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
